### ISSUE TITLE
"The sum of all fears" does not start.

### ISSUE THREAD

**Issue by aprats1** [NVIDIAGameWorks/rtx-remix#46]
**Describe the bug**
- A clear and concise description of what the bug is.

After placing all the files from the remix-0.1.1 directory and the d3d8.dll file next to the game executable and trying to start it normally, the game does not start. Instead, an error message appears (the screenshot shows the error in Spanish, I will translate it to English):

"The entry point of the procedure is not found MiniDumpWriteDump in the dynamic link library .D:\Redstorm Entretainment\Panico Nuclear\d3d9.dll"

- Which Remix version (include branch and commit info for dxvk and bridge if not a stable build)

remix-0.1.1

- Which GPU

RTX 350TI 4gb Laptop

- Which GPU driver version

531.41

- Which game build (and whether it is from Steam/GOG/Other)

CD version. Build 2002 Spanish version

**Attach logs!**
- NvRemixBridge.log
- d3d9.log
- The game specific log will be named gamename_d3d9.log

**To Reproduce**
1. Add files from the remix-0.1.1 folder and the d3d8.dll file to D:\Red Storm Entertainment\Pánico Nuclear (for English versions: D:\Red Storm Entertainment\SOAF D:\Red Storm Entertainment\Sum Of All Fears)
2. Click on 'SOAF.exe'
3. See error

**Expected behavior**

The game should start and the new Nvidia remix options should be visible working with the game.

**Screenshots**

![image](https://user-images.githubusercontent.com/37695521/231736455-fa085a46-372c-4d6c-b51e-4a5ba13c639f.png)

**Comment by aprats1** [comment]
Deleting the **dgbhelp** file seems to help, now the game starts but I can only make it start in windowed mode. It also starts in fullscreen mode, but the resolution is stuck and I can only see about a quarter of the image.

I managed to start a game, but I haven't seen any changes in the game. I managed to add most of the textures, but I haven't seen the RTX working at any point. I think I need to configure more things, I will keep investigating but if anyone can give me a hint, I would be grateful.
